template-vue-ts/package.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "vite-vue-typescript-starter",
+  "version": "0.0.0",
+  "scripts": {
+    "dev": "vite",
+    "build": "vite build"
+  },
+  "dependencies": {
+    "vue": "^3.0.0-beta.14"
+  },
+  "devDependencies": {
+    "@vue/compiler-sfc": "^3.0.0-beta.14",
+    "@vuedx/typescript-plugin-vue": "^0.1.0-alpha.12",
```

Comment:
May I know what this plugin provides and how it's different than [this](https://github.com/vuejs/vue-cli/tree/dev/packages/@vue/cli-plugin-typescript#readme)?

Reply:
This plugin support vue 3.